src/Card/CardImageCap.jsx

```diff
@@ -47,24 +48,29 @@ const CardImageCap = React.forwardRef(({
 
   const handleSrcFallback = (event, altSrc) => {
     const { currentTarget } = event;
-    if (currentTarget.src !== altSrc) {
+
+    if (!altSrc || currentTarget.src.endsWith(altSrc)) {
+      currentTarget.style.display = 'none';
+    } else if (currentTarget.src !== altSrc) {
       currentTarget.src = altSrc;
     }
   };
 
   return (
     <div className={classNames(className, wrapperClassName)} ref={ref}>
       <img
-        className="pgn__card-image-cap"
+        className={classNames('pgn__card-image-cap', { show: show.imageCap })}
         src={src}
         onError={(event) => handleSrcFallback(event, fallbackSrc)}
+        onLoad={() => setShow({ ...show, imageCap: src })}
```

Comment:
[suggestion] If this new state variable is initialized with boolean values, we should keep the data types consistent. In this line, it looks like the string `src` is being passed instead of a true boolean data type.